README.md

```diff
@@ -162,17 +162,17 @@ Set the current value for your component. If only a single number is provided, o
 
 If you want to work on this project locally, you need to grab all of its dependencies.
```

Comment:
Maybe add a line saying, recommend using `yarn` when you want to work on this project. And provide a link with instructions on how to set up yarn.